packages/pkg/src/index.ts

```diff
@@ -39,10 +43,25 @@ export default {
         const { filepath } = options
         const ast = parse(text, options) as { node: ObjectExpression }
         if (PKG_REG.test(filepath)) {
-          ast.node.properties = format(ast.node.properties)
+          ast.node.properties = format(
+            ast.node.properties,
+            options as FormatOptions,
+          )
         }
         return ast
       },
     },
   },
+  options: {
+    packageSortOrder: {
+      since: '0.21.0',
+      category: 'Package',
+      type: 'string',
+      array: true,
+      default: [{ value: [] }],
+      description:
+        'An array of property names to sort the package.json properties by.',
+    },
+  },
+  defaultOptions: {},
```

Comment:
And this?

Reply:
I believe we don't need this, I never used it in other plugins like `sh` and `sql`.

Reply:
```suggestion
```